cliquet/resource.py

```diff
@@ -416,16 +425,10 @@ def create_record(self, record):
                 record['index'] = idx
                 return record
 
-        :raises: :exc:`~pyramid:pyramid.httpexceptions.HTTPConflict`
-            if a unique field constraint is violated.
-
         :returns: the newly created record.
         :rtype: dict
         """
-        try:
-            return self.db.create(record=record, **self.db_kwargs)
-        except storage_exceptions.UnicityError as e:
-            self._raise_conflict(e)
+        return self.db.create(record=record, **self.db_kwargs)
```

Comment:
Ah no because it depends on a PR waiting review/merge in Kinto https://github.com/mozilla-services/kinto/pull/32,  itself waiting for https://github.com/mozilla-services/kinto/pull/27